modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogHashIndexDescriptorSerializers.java

```diff
@@ -63,4 +63,33 @@ public void writeTo(CatalogHashIndexDescriptor descriptor, IgniteDataOutput outp
             writeStringCollection(descriptor.columns(), output);
         }
     }
+
+    @CatalogSerializer(version = 1, since = "3.1.0")
```

Comment:
Here and other newly added serializers version should be equal to 2.
```suggestion
    @CatalogSerializer(version = 2, since = "3.1.0")
```

Reply:
Thanks. Fixed.